test/rw.jl

```diff
@@ -346,14 +349,7 @@ function checkexpr(a::Expr, b::Expr)
     @assert i >= length(a.args) && j >= length(b.args)
 end
 
-fn = joinpath(tempdir(),"test.jld")
-
-# Issue #106
```

Comment:
Yes. It was dead code and I don't think the original JLD issue should apply to the implementation of type lookup here.